distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1467,9 +1467,9 @@ type Table
        - case_sensitivity: Specifies if the text values should be compared case
          sensitively.
        - on_problems: Specifies the behavior when a problem occurs.
```

Comment:
The doc comment explaining the added `at_least_one` argument should be replicated here too for consistency.